### DISCUSSION TITLE
Is it possible to highlight some links by specific rule in the editor?

### DISCUSSION THREAD

**Discussion by demo-igor** [WordPress/gutenberg#34534] [Q&A] [answered]
I would like to highlight external links in the editor but do not see any way to do it. Could someone helps to understand is it possible at all?

**Comment by Mamaduka** [comment, marked as answer]
Hi, @demo-igor

You can probably do this via CSS. Here's an article explaining some methods - https://davidwalsh.name/external-links-css.

**Reply by demo-igor** [reply]
@Mamaduka thanks. I've already did it with the css

**Reply by Mamaduka** [reply]
Happy to hear that. In that case, I'm going to mark this question as resolved.
